### DISCUSSION TITLE
Chinese character 舃 appearing after @ in Commit messages using renovate 38.44.2

### DISCUSSION THREAD

**Discussion by ceisele-r** [renovatebot/renovate#30911] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub - renovatebot/github-action@40.2.6 with Renovate v38.44.2

### Please tell us more about your question or problem

I think that the change from #30842 causes strange Chinese characters to appear in commit messages (displayed in Github and TortoiseGit) as I am now facing such commit messages in TortoiseGit
![image](https://github.com/user-attachments/assets/6b075baf-2556-4111-af87-3edfedd60f10)
and Github
![image](https://github.com/user-attachments/assets/b5e4c66a-2e83-458e-99c2-9a4dfd976693)
using renovate 38.44.2 (via renovatebot/github-action).

I guess it's because the zero-width space is interpreted as "舃" in these locations.

It happens using one of our internal packages (npm, scoped to `@ourorg/ourpackage`) but I don't think this is related to the specific package as the package was updated multiple times before correctly (with "correct" commit message) using a prior renovate version (38.40.1).

### Logs (if relevant)

<details><summary>No Logs</summary>

```

It seems the commit message is not logged in the logs

```

</details>

**Comment by rarkins** [comment]
Should be fixed by https://github.com/renovatebot/renovate/pull/30842

**Reply by ceisele-r** [reply]
Sorry, I'm a bit confused. Did you mean this should be fixed by the revert #30910 ?
Or by the actual change #30842 (which I suspect to cause the issue)?

**Reply by HonkingGoose** [reply]
I think this is fixed by _reverting_ to the old state with PR:

- #30910

**Reply by ceisele-r** [reply, marked as answer]
Yeah, how I wrote it's fixed by #30910 .
